examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace CONFID with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/CONFID.json">
```

Comment:
... And actually, if you have a remote configuration, all of the vendor configuration you're specifying here can be included there instead. It should work either way, but just curious if there's a particular reason you split the config so that some of it comes via `config="config-url"` and some via `type="krux"`.